demos/tests/cases.py

```diff
@@ -50,6 +52,8 @@ def models_lst_path(self, source_dir):
     def device_args(self, device_list):
         if len(self.device_keys) == 0:
             return {'CPU': []}
+        if self.supported_devices:
+            device_list = self.supported_devices
```

Comment:
Should this be an intersection of `device_list` and `self.supported_devices`?

Reply:
Updated